src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -177,6 +177,15 @@ public void delete() {
     Context.setWorkspace(null);
   }
 
+  /**
+   * Enable the current user to impersonate their pet SA in the current workspace.
```

Comment:
same as above, enables current user and their pet to impersonate the pet, I think